tools/bpf2c/templates/kernel_mode_bpf2c.vcxproj

```diff
@@ -40,6 +40,9 @@
     <RootNamespace>$(FileName)</RootNamespace>
     <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
     <ProjectName>$(FileName)_km</ProjectName>
+    <!-- Required for Microsoft internal OneBranch builds -->
+    <UseInternalMSUniCrtPackage>true</UseInternalMSUniCrtPackage>    
+    <UndockedKernelModeBuild>true</UndockedKernelModeBuild>
```

Comment:
Why here instead of some Directory.Build.props in whatever place the internal mirror is?

Reply:
Specifically, I **cannot** build XDP completely in an official sandboxed, OneBranch pipeline because of this right now.

Reply:
I was confused by the PR description that sounded like this was needed for an internal mirror rather than a onebranch build in github.

Reply:
Sorry. While the build does happen internally, the source does not necessarily have to be internal.

Reply:
Discussed that the public github ci/cd for xdp-for-windows will eventually hit this, so approved after "internal" was removed from the comment.